src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationDashboard/requestLog.tsx

```diff
@@ -55,8 +55,10 @@ const getEventTypes = memoize((app: SentryApp) => {
 
   const events = [
     ALL_EVENTS,
-    'installation.created',
-    'installation.deleted',
+    // Internal apps don't have installation webhooks
+    ...(app.status !== 'internal'
```

Comment:
@nolachen wish we could make this look cleaner

Reply:
@scefali Another way I could think of is to send the event types that do exist from the backend, so that they would be in sync. But then we would not show "placeholders" for event types that are possible but don't have any data